examples/warehouse/domain/example.com/dns.yaml

```diff
@@ -23,3 +23,8 @@ net:
         protocol: tcp
         port: 5269
         target: xmpp2
+    txt:
+      - label: example.com.
```

Comment:
Repeating the domain FQDN as label is not DRY, i think we should use explicit YAML `~` or leave the label unspecified to specify a "null" label.

Reply:
A `~` shows up as an empty string (which is probably yet another part of #135), but leaving the label unspecified works.